dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/dragen/AnalysisFile.java

```diff
@@ -0,0 +1,101 @@
+package ca.on.oicr.gsi.runscanner.dto.dragen;
+
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+// Represents one file output by DRAGEN
+public class AnalysisFile {
+  private Path path;
+  private String crc32Checksum;
+  private long size;
+  private Instant createdTime;
+  private Instant modifiedTime;
+
+  // What will be in this map will be unique to the particular DRAGEN Workflow
+  private Map<String, Object> info = new HashMap<>();
```

Comment:
Jackson won't be able to deserialize `Object` and there should be strictly defined fields anyway. Any workflow that needs its own fields should have a subclass of AnalysisFile. Looks like BCLConvert needs read count and read number which are currently being added to this as "read_count" and "read_number" in `ProcessDragen`, which is also inconsistent formatting with the rest of Run Scanner's output